nixos/modules/virtualisation/virtualbox-image.nix

```diff
@@ -18,6 +18,13 @@ in {
           The size of the VirtualBox base image in MiB.
         '';
       };
+      baseImageFreeSpace = mkOption {
```

Comment:
If this is the free space, what is `baseImageSize`? What significance do those 50GiB have on the end result?

Reply:
well baseImageSize is the imagesize of the image. which is currently "auto" so it is as big as it needs to be to fit everything inside. I guess there should be no significance. since the baseImageSize was 50 * 1024 before.

Reply:
Oh I misread, right, `"auto"` is the default, not 50GiB